### PR TITLE
Adds two new roles: reviewer and mentor

### PR DESCRIPTION
**Pull request by kbraak**
Kindly review the definitions of these two new agent roles - thank you!

### CODE REVIEW

File: vocabulary/gbif/agent_role.xml [hunk 1]
```diff
@@ -219,4 +219,28 @@
...
+      <term dc:title="Verificador" xml:lang="es"/>
+      <term dc:title="Vérificateur" xml:lang="fr"/>
+      <term dc:title="審閱者" xml:lang="zh"/>
+    </preferred>
+    <alternative>
+    </alternative>
+  </concept>
+  <concept dc:identifier="mentor" dc:URI="http://rs.gbif.org/vocabulary/gbif/agentRole/mentor"
+           dc:relation=""
+           dc:description="an agent who is highly experienced in publishing biodiversity data and who has provided guidance and constructive feedback during the publishing process.">
```
Comment (mdoering) [inline vocabulary/gbif/agent_role.xml]: does it have to be biodiversity data?

Comment (kbraak) [inline vocabulary/gbif/agent_role.xml]: Ideally a suitable mentor would have been awarded a biodiversity publishing badge for example. I didn't want to limit it to only mentors with badges though..

**Comment by peterdesmet** [comment]
This roles describes the role I take often in publishing data, but is there a big difference between the two? Which one of the two is actively involved in the publication?

Also, I wouldn't know how to define "certified" and "all quality requirements".

**Comment by kbraak** [comment]
Thanks Peter, 

It certainly sounds like these roles could raise the visibility of your efforts in helping publish others' datasets. 

The mentor would be more actively involved in publishing. That is because in order to share their skills and knowledge, they may have to demonstrate how to use the IPT for example. 

The reviewer, on the other hand, only performs quality control checks on the dataset and relays feedback to the publisher. They should never have to demonstrate how to use the IPT.

A checklist listing all quality requirements for the dataset is currently under development: https://github.com/gbif/ipt/wiki/dataQualityChecklist The reviewer would use such a checklist to ensure the dataset fulfills all quality requirements. Naturally your feedback on this checklist would be greatly appreciated, but perhaps better done through mail. Thanks.

**Comment by siro1** [comment]
I understand that the reviewer role here is analogous to the role played by peer reviewers in journal and book publication processes.

**Comment by kbraak** [comment]
Thank you @siro1. Based on your comment, would you like to improve the definition as follows? 

reviewer is:

> "a certified agent responsible for examining the dataset and providing assurance that it fulfills all quality requirements defined by GBIF. This role is analogous to the role played by peer reviewers in journal and book publication processes."

**Comment by peterdesmet** [comment]
That addition helps, but a reviewer can not provide assurance that quality req are met. He/she can only point out where it fails. Also, would not use the word "agent" (too technical) or "certified" (unclear by whom). My suggestion:

> "person assigned to review the dataset and verify that it fulfills all quality requirements defined by GBIF. This role is analogous to the role played by peer reviewers in the scholarly publication process."

**Comment by siro1** [comment]
Simplicity is always better so I like the edits by Peter. Here is a revised version done in consultation with Kyle.

"person assigned to review the dataset and check if it fulfills all quality requirements defined by GBIF. This role is analogous to the role played by peer reviewers in the scholarly publication process. Ideally the reviewer has attained a GBIF assigned **biodiversity data reviewer badge**"

**Comment by siro1** [comment]
Following on the revised text for reviewer, here is the revised one for mentor:

"a person who is highly experienced in publishing biodiversity data and who has provided guidance and constructive feedback during the publishing process. Ideally the mentor has attained a GBIF assigned training badge, onsite support badge or helpdesk support badge"

**Comment by mdoering** [comment]
What impact do new roles have to GBIF code?
Should they also be added to https://github.com/gbif/gbif-api/blob/master/src/main/java/org/gbif/api/vocabulary/ContactType.java? Anything else?

**Comment by laurarussell** [comment]
@mdoering -- @melianieraymond and I just discussed. We don't think mentor role is necessary. Reviewer is okay, but we would take out the mention of badges.  Here is revised text.

"person who has reviewed the metadata and/or the data to verify if the published resource satisfies the quality requirements defined by GBIF. This role is analogous to the role played by peer reviewers in the scholarly publication process."

**Comment by mdoering** [comment]
Thanks Laura, I prefer that too. I even wonder if GBIF should be mentioned at all. Its not mentioned in any other role so far: https://github.com/gbif/rs.gbif.org/blob/master/vocabulary/gbif/agent_role.xml

**Comment by mdoering** [comment]
How about 

> person assigned to review the dataset and verify its data quality. This role is analogous to the role played by peer reviewers in the scholarly publication process.

**Comment by peterdesmet** [comment]
Yes please, without reference to GBIF

**Comment by laurarussell** [comment]
@mdoering we did want to have a piece in there about the metadata because for data papers reviewers are specifically reviewing the metadata. So it should be more than just data quality. Fine with removing the GBIF part.

**Pull request merged**